Documentation/sources/enum-public-classes/class-message.md

```diff
@@ -63,35 +63,6 @@ void setAutoTrack(boolean enabled)
 
 * *enabled* - if true, Experience Edge events will automatically be generated when this `Message` is triggered, displayed, or dismissed.
 
-### evaluateJavascript
```

Comment:
The `dismiss` method of this class no longer accepts an argument. Could you also please update that?
```
### dismiss
Signals to the UIService that the message should be removed from the UI.
```java
void dismiss()
```
```